api/errors.go

```diff
@@ -0,0 +1,7 @@
+package api
+
+import "errors"
+
+var (
+	ErrMethodNotAllowed = errors.New("method not allowed")
+)
```

Comment:
This is currently unused, I wonder if the idea is to hold these errors VS exposing the error from the "controller" methods:
- https://github.com/replicatedhq/embedded-cluster/blob/063d0e401a65fbb74ae36add1ed552010dbcaef2/api/console/api.go#L45-L49

I also believe we should be returning the error msg under a json payload like - `{"error": "Config is broken #sorrynotsorry"}` - to keep consistency with our clients.